client/components/domains/registrant-extra-info/docs/example.jsx

```diff
@@ -7,13 +7,16 @@ import React, { PureComponent } from 'react';
  * Internal dependencies
  */
 import Card from 'components/card';
-import ExtraInfoFrForm from 'components/domains/registrant-extra-info/fr-form';
+import FrForm from 'components/domains/registrant-extra-info/fr-form';
+import CaForm from 'components/domains/registrant-extra-info/ca-form';
 import { forDomainRegistrations as getCountries } from 'lib/countries-list';
 const countriesList = getCountries();
 
 class ExtraInfoFrFormExample extends PureComponent {
 	state = {
-		registrantExtraInfo: {},
+		registrantExtraInfo: {
+			extra: {},
+		},
```

Comment:
Our levels of nesting are starting to feel a little funky to me. Can we mask the second layer and inject that structure during action creation or something?  I'll think and play with this more tonight.